src/airport.h

```diff
@@ -181,6 +181,22 @@ struct AirportFTAClass {
 	byte nofelements;                     ///< number of positions the airport consists of
 	const byte *entry_points;             ///< when an airplane arrives at this airport, enter it at position entry_point, index depends on direction
 	byte delta_z;                         ///< Z adjustment for helicopter pads
+
+	/**
+	 * Get the number of terminals at the airport.
+	 * @param apc Airport description.
+	 * @return Number of terminals.
+	 */
+	uint GetNumTerminals() const
+	{
+		uint num = 0;
+
+		if (terminals != nullptr) {
+			for (uint i = terminals[0]; i > 0; i--) num += terminals[i];
+		}
+
+		return num;
+	}
```

Comment:
What purpose does making this a member function serve?

Reply:
Seemed to be the best way to go about it, now that it is called from two places.

Reply:
I just moved it away from the FTA to somewhere else.